src/Syntax.hs

```diff
@@ -97,18 +93,18 @@ data Expr
 --    | Look Pn Id [Expr]
     | Func Pn Id [Expr]
     | ListConst Expr
-    | EmptyList
+    | EmptyList Pn
```

Comment:
This is unused atm. Do we need it @MrChico?

Reply:
lets move it right now